src/main/java/com/hoverla/bibernate/session/ResultSetParser.java

```diff
@@ -0,0 +1,49 @@
+package com.hoverla.bibernate.session;
+
+import com.hoverla.bibernate.annotation.Column;
+import com.hoverla.bibernate.exception.ResultSetException;
+
+import java.lang.reflect.Field;
+import java.sql.ResultSet;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class ResultSetParser {
```

Comment:
I have the same class in my review, so we need to decide which one we will use. I have covered my class with tests, so I propose to use my one if it's not break anything and you don't mind.